defaults/main.yml

```diff
@@ -14,23 +16,31 @@ fluentbit_svc_name: "td-agent-bit"
 fluentbit_svc_flush: "5"
 fluentbit_svc_grace: "5"
 fluentbit_svc_daemon: "off" # Need to be Off when using the provided systemd unit.
-fluentbit_svc_logfile: ""
-fluentbit_svc_loglevel: "info"
+fluentbit_svc_loglevel: 'error'
+fluentbit_log_path: '/var/log/td-agent-bit'
+fluentbit_svc_logfile: '/var/log/td-agent-bit/td-agent-bit.log'
 fluentbit_svc_parsers:
-  - parsers.conf
-fluentbit_svc_plugins:
-  - plugins.conf
+  - custom_parsers.conf
+fluentbit_svc_plugins: []
 fluentbit_svc_streams: []
 
-fluentbit_svc_http:
-  server: "off"
+# Storage
+fluentbit_dbs_path: '/var/run/fluentbit/'
 fluentbit_svc_storage:
-  metrics: "on"
+  Path: '/var/run/fluentbit/fluentbit.buffer'
+  Sync: normal
+  Backlog.mem_limit: '10M'
```

Comment:
Is upper case mandatory? I find upper case variables error prone

Reply:
Nope, corrected